nautobot/dcim/api/serializers.py

```diff
@@ -1015,7 +1017,8 @@ class Meta:
         opt_in_fields = ["computed_fields"]
 
 
-class InterfaceSerializer(
+# TODO: collapse this with InterfaceSerializer in 2.0.
+class InterfaceSerializerVersion13(
```

Comment:
I think this would better be named as `Version12` as it's in use by both the v1.2 and v1.3 REST API?